bundles/binding/org.openhab.binding.myq/src/main/java/org/openhab/binding/myq/internal/MyqData.java

```diff
@@ -82,16 +85,14 @@ public GarageDoorData getMyqData() {
 	 *            Attempt number when it recursively calls itself
 	 */
 	private String getGarageStatus(int attemps) {
-		if (this.sercurityTokin == null && Login()) {
+		if (sercurityTokin == null && !Login()) {
```

Comment:
So this was originally if (this.sercurityTokin == null && Login())  I think you want "if not null and login fails" ? Make sure I changed this correctly.